src/components/Match/matchColumns.jsx

```diff
@@ -51,7 +51,26 @@ export const heroTdColumn = {
 
 const partyStyles = (row, match) => {
   if (match.players && match.players.filter(x => x.party_size > 1).length) {
-    return <div className={`group${row.party_id}`} />;
+    const groupedPlayers = lodash.compose(
```

Comment:
can we remove these lodash references?

Reply:
Would you prefer I Implement groupBy by hand? Or is there another lib you had in mind?

Reply:
Yes, please, if possible.  You could just copy the lodash implementation (if it doesn't have any dependencies)

Reply:
Okay. I will get to it tomorrow. Just out of curiosity, is there no plan to pull in any utility library? Things might get out of hand if people are copy-pasting lodash functions all the time.

Reply:
Generally we've found that with the latest ES6 features most things can be implemented nicely with native functions.  Any utility functions we need can be added to utility.js for reusability.